vendor/vendor.json

```diff
@@ -126,6 +126,24 @@
 			"revision": "af9db2027c98c61ecd8e17caa5bd265792b9b9a2",
 			"revisionTime": "2018-03-18T00:15:26Z"
 		},
+		{
+			"checksumSHA1": "cEszpxh1szqTb440ze4hm/Vfm40=",
+			"path": "github.com/coreos/go-systemd/sdjournal",
```

Comment:
Seeing that there are release for go-systemd, I wonder if we should tie it to one?